packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+# Slider
+
+## Component Description
+
+The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
```

Comment:
FUIR9 only supports single value. The range option is not in the Figma spec.

Reply:
If it's there now but wasn't before, let's (for now) take is a follow up given I'm not sure that range exists in FAST at this time either.

Reply:
I'm good with kicking this down the road until the design spec is more fleshed out, but it looks like it does exist in [the FAST Slider attributes](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/slider/slider.spec.md#api).